packages/vscode-extension/src/handlers/copilotChatHandlers.ts

```diff
@@ -241,11 +241,36 @@ export async function invokeTeamsAgent(args?: any[]): Promise<Result<null, FxErr
   const triggerFromProperty = getTriggerFromProperty(args);
   ExtTelemetry.sendTelemetryEvent(TelemetryEvent.InvokeTeamsAgentStart, triggerFromProperty);
 
-  const query =
-    triggerFromProperty["trigger-from"] === TelemetryTriggerFrom.TreeView ||
-    triggerFromProperty["trigger-from"] === TelemetryTriggerFrom.CommandPalette
-      ? "@teamsapp Use this GitHub Copilot extension to ask questions about Teams app and agent development."
-      : "@teamsapp Write your own query message to find relevant templates or samples to build your Teams app and agent as per your description. E.g. @teamsapp create an AI assistant bot that can complete common tasks.";
+  let query = "";
+  switch (triggerFromProperty[TelemetryProperty.TriggerFrom]) {
+    case TelemetryTriggerFrom.TreeView:
+      query =
+        "@teamsapp Use this GitHub Copilot extension to ask questions about Teams app and agent development.";
+      break;
+    case TelemetryTriggerFrom.CommandPalette:
```

Comment:
can these two conditions be put together?

Reply:
Yes, I will put them together. Thanks!